src/components/Table/Table.scss

```diff
@@ -50,6 +50,9 @@
     }
 
     &__cell {
+        position: relative;
+
+        min-width: 17px;
```

Comment:
Let's put these styles to the separate class. Class can be passed in enhanceColumns method

Reply:
[separete](https://github.com/gravity-ui/uikit/pull/654/commits/7912e104afbadd045a107d072087d0f60272a472#diff-934636fcc01112fd9d8f251ed6903afc4d5a1b202f44f1828abd5b1f4e07e925R4) class

[use](https://github.com/gravity-ui/uikit/pull/654/commits/7912e104afbadd045a107d072087d0f60272a472#diff-e550fc4a251be96df10c117d97a0b9525d3ed80d4177be7d2169fa7518e4fa0cR165) in `enhanceColumns` method